common/namespace/replication_task_executor.go

```diff
@@ -130,8 +130,10 @@ func (h *namespaceReplicationTaskExecutorImpl) shouldProcessTask(ctx context.Con
 	})
 	switch err.(type) {
 	case nil:
-		if resp.Namespace.Info.Id != task.GetId() {
-			h.logger.Error("namespace replication encountered name / UUID collision during determination on whether to process task", tag.WorkflowNamespaceID(task.Info.GetName()))
+		if resp.Namespace.Info.GetId() != task.GetId() {
```

Comment:
should we change to use `resp.Namespace.Info.GetId()` in other place as well?

Reply:
same thing. I am reverting this one.